massa-metrics/Cargo.toml

```diff
@@ -9,7 +9,8 @@ sandbox = []
 
 [dependencies]
 prometheus = {workspace = true, "features" = ["process"]}
-hyper = {workspace = true, "features" = ["server", "http1"]}   # BOM UPGRADE     Revert to {"version": "0.14.26", "features": ["server", "tcp", "http1"]} if problem
+# hyper = {workspace = true, "features" = ["server", "http1"]}   # BOM UPGRADE     Revert to {"version": "0.14.26", "features": ["server", "tcp", "http1"]} if problem
```

Comment:
comment :)

Reply:
Should keep this one as we need a quick way to switch between "hyper workspace" and "hyper local version"